examples/example.krun

```diff
@@ -1,6 +1,11 @@
 import os
 from krun.vm_defs import (PythonVMDef, JavaVMDef)
 from krun import EntryPoint
+from distutils.spawn import find_executable
+
+# For a real experiment you would certainly use absolute paths
+PYTHON27_BIN = find_executable("python2.7")
```

Comment:
Having these as top-level variables is obfuscatory, since they're only referenced in one place. Why not say `'vm_def' : JavaVMDef(find_executable("python2.7"))`?